docs/django.md

```diff
@@ -12,18 +12,16 @@ weight=4
 
 ## Quickstart Guide: Compose and Django
 
-
-This Quick-start Guide will demonstrate how to use Compose to set up and run a
+This quick-start guide will demonstrate how to use Compose to set up and run a
 simple Django/PostgreSQL app. Before starting, you'll need to have
 [Compose installed](install.md).
 
 ### Define the project
 
-Start by setting up the three files you'll need to build the app. First, since
+For this project, you need to define three files. First, since
 your app is going to run inside a Docker container containing all of its
-dependencies, you'll need to define exactly what needs to be included in the
-container. This is done using a file called `Dockerfile`. To begin with, the
-Dockerfile consists of:
+dependencies, you'll need a `Dockerfile` to define exactly what needs to be
+included in the image. To begin with, create a `Dockerfile` consists of:
```

Comment:
"image" is arguably a bit more technically accurate, but for new users, "container" might be less confusing (and it's not wrong).